cms/envs/test.py

```diff
@@ -136,7 +136,8 @@
     MIGRATION_MODULES = NoOpMigrationModules()
 
 LMS_BASE = "localhost:8000"
-FEATURES['PREVIEW_LMS_BASE'] = "preview"
+FEATURES['PREVIEW_LMS_BASE'] = "preview.localhost"
```

Comment:
@dianakhuang In bokchoy tests, there is an actual lms server running which the browser needs to connect to. Unit tests need some dummy value because there are no requests over http.